fortitude/src/rules/mod.rs

```diff
@@ -82,6 +82,7 @@ pub fn code_to_rule(category: Category, code: &str) -> Option<(RuleGroup, Rule)>
         (Style, "061") => (RuleGroup::Stable, Ast, style::end_statements::UnnamedEndStatement),
         (Style, "071") => (RuleGroup::Preview, Ast, style::double_colon_in_decl::MissingDoubleColon),
         (Style, "101") => (RuleGroup::Stable, Text, style::whitespace::TrailingWhitespace),
+        (Style, "121") => (RuleGroup::Preview, Ast, style::whitespace::IncorrectSpaceBeforeComment),
```

Comment:
We're not very consistent with our rule codes, but should this be 111?

Reply:
I'm not sure. I honestly just took a guess that there might be 20-ish whitespace rules we might want to add before this one. Maybe just 102 would make more sense?

Reply:
Yeah, I better there's a few whitespace rules! Let's for for 102 then

Reply:
Fixed in https://github.com/PlasmaFAIR/fortitude/pull/194/commits/ad6455db1917dc15fc752f00757a1ea9e83a8f73